test/labeling/apply/test_lf_applier.py

```diff
@@ -84,3 +87,26 @@ def test_lf_applier_pandas_spacy_preprocessor(self) -> None:
         applier = PandasLFApplier([first_is_name, has_verb])
         L = applier.apply(df)
         np.testing.assert_equal(L.toarray(), L_TEXT_EXPECTED)
+
+
+class TestDaskApplier(unittest.TestCase):
+    def test_lf_applier_dask(self) -> None:
+        df = pd.DataFrame(dict(num=DATA))
+        df = dd.from_pandas(df, npartitions=2)
```

Comment:
Nope! It just corresponds to the number of parallel processes we'll run. 2 is a good test here because the partitions will be unequal sizes (2 rows and 1 row). We want to keep it at minimal parallelism here because the Travis machines aren't very beefy (like maybe 2 cores) and want to avoid CPU oversubscription.

Reply:
Yeah looks like just 2 cores: https://docs.travis-ci.com/user/reference/overview/#virtualisation-environment-vs-operating-system. I think we're ok just doing `npartitions=2`, thoughts?